snippets/country-localization.liquid

```diff
@@ -13,10 +13,12 @@
     aria-controls="{{ localPosition }}List"
     aria-describedby="{{ localPosition }}Label"
   >
-    <span>
-      {{- localization.country.currency.iso_code }}
-      {{ localization.country.currency.symbol }} | {{ localization.country.name -}}
-    </span>
+    
+    <span  class="localization-form__country_flag">
+      {{ localization.country | image_url: width: 18 | image_tag }} 
```

Comment:
I think you could also have the image outside of the span. The container (button) is already a display flex with `align-items: center;` 🤔 

That prevents us from adding more CSS 👍

Reply:
Done!